orchestration/dagster_orchestration/hca_orchestration/resources/__init__.py

```diff
@@ -13,5 +15,8 @@
     local_beam_runner,
     local_storage_client,
     noop_data_repo_client,
-    test_beam_runner
+    test_beam_runner,
+    noop_bigquery_client,
+    bigquery_client,
+    load_tag
```

Comment:
do we think we're reaching a point where shoving these into the top-level class is getting cumbersome? when do we think we'll reach that point, if not?

Reply:
🤔 So, I foresee us having the following resources:
* Slack
* Load tag
* BigQuery (staging area)
* BigQuery (jade) (this may be redundant if we use the jade project ID at runtime instead of baking it at init time)
* Jade API client
* Beam runners
* Google storage

Each of those will have different flavors depending on the mode. So, yeah, to your point, I think this is an opportune time to refactor. Perhaps we can split them into a hierarchy of environment packages?